web/add/dns/index.php

```diff
@@ -276,7 +276,7 @@
 	// Flush field values on success
 	if (empty($_SESSION["error_msg"])) {
 		$_SESSION["ok_msg"] = sprintf(
-			_("Record {%s.%s} has been created successfully."),
+			_("Record <b>%s.%s</b> has been created successfully."),
```

Comment:
Don't use html in the translations

{%s.%s}

https://github.com/hestiacp/hestiacp/blob/4b2401a78d19d225c212b6facb0a7962c1797bfa/web/inc/i18n.php#L41-L52

See for a working example:
https://github.com/hestiacp/hestiacp/blob/5f19361c85072f3368fbf63ab455488ac45107c9/web/add/db/index.php#L195-L200

Reply:
Has been revert.